pkg/selector/outputs/verboseView.go

```diff
@@ -29,16 +29,14 @@ const (
 	outlinePadding = 8
 
 	// controls
-	verboseControls = "Controls: ↑/↓ - up/down • enter - return to table • q - quit"
+	verboseControls = "Controls: ↑/↓ - up/down • e - return to table • q - quit"
```

Comment:
What was the reason to change this? Did this change from expecting an "enter/return" keypress to an "e"?

Also, should we have something besides "e" to indicate the return to the previous view? Such as the "esc" used in the `sortingView`?

Reply:
When I first created the expanded verbose view, there were no other controls so I thought that "enter" was a reasonable key, but now that we have controls for filtering (f), sorting (s), and trimming (t), the controls for each function use a key with the same starting letter as the feature's name, so I found it best to also change the control for "expanding" to e. 

Although "e" makes sense in table view, it does not make sense when exiting the expanded verbose view, so I do feel as though "esc" would be a better key to be used when transitioning from the verbose view to the table view.

Reply:
Changed "e" to "esc" here: [97c6569](https://github.com/aws/amazon-ec2-instance-selector/pull/152/commits/97c65695b87149a32cb71937dcbbb248076db6e3)